packages/block-editor/src/components/block-list/block-wrapper.js

```diff
@@ -98,6 +98,10 @@ const BlockComponent = forwardRef(
 		 * When a block becomes selected, transition focus to an inner tabbable.
 		 */
 		const focusTabbable = () => {
+			if ( ! document.activeElement ) {
```

Comment:
I'm not sure I understand this change, if we select a block, we still want it to be focused regardless of whether there was an activeElement before?

Reply:
That's a good point. I think I was misled by the early `return;` of the condition following this one. Maybe instead, it should be this?

```js
if (
	document.activeElement && isInsideRootBlock( wrapper.current, document.activeElement )
) {
	return;
}
```

Reply:
makes sense yes

Reply:
cc607cf53a